action.yml

```
@@ -1,22 +1,29 @@
-name: 'TeePublic pull-ECS-task-and-update-ENV-vars Action for GitHub Actions'
-description: 'Update a task definition with ENV vars from Parametere Store in an Amazon ECS task definition JSON file, creating a new file.'
-# branding:
-#   icon: 'cloud'
-#   color: 'blue'
+name: 'TeePublic ECS Deploy Helper'
+description: 'Determine which docker tag we want to deploy + Get current task definition + Fill in current environment variables from parameter store'
 inputs:
-  task-definition:
-    description: 'The path to the ECS task definition JSON file'
+  provided-image-tag:
+    description: 'Optionally provided image tag for override deploys'
     required: true
   service-name:
     description: 'The name of the service, such as "nginx" or "search-admin"'
     required: true
+  aws-region:
+    description: 'The AWS region to use with API requests'
+    required: true
```

```suggestion
    default: us-east-1
```

Since our workloads are – for now – in us-east-1, how would you feel about defaulting this value?